src/app/content/search/utils.ts

```diff
@@ -147,11 +148,18 @@ export const highlightResults = (
     }
 
     const hitHighlights = hit.highlight.visibleContent.map((highlightText, index) => {
-      const highlights = getHighlightRanges(element, highlightText).map((range) => {
-        const highlight = new Highlight(range.nativeRange, {content: range.toString()});
-        highlighter.highlight(highlight);
-        return highlight;
-      });
+      const highlights = getHighlightRanges(element, highlightText)
+        .map((range) => {
+          const highlight = new Highlight(range.nativeRange, {content: range.toString()});
+
+          attachHighlight(highlight, highlighter, (failedHighlight) =>
+            `Search result highlight with id: ${failedHighlight.id} has not been attached`
```

Comment:
Maybe @TomWoodward can decide if this message is okay and if not then what data should you pass to make it more clear

Reply:
the id is not useful because for search it will be random. the useful info would be the search term and the page we're on, if you can easily get that in there.

if not, just put a fixed string like "Search result failed to highlight" and we might be able to get the other relevant data out of sentry if we try hard enough